tests/sles4sap/qesapdeployment/test_cluster.pm

```diff
@@ -30,6 +31,10 @@ sub run {
     qesap_ansible_cmd(cmd => $_, provider => $prov) for @remote_cmd;
     qesap_ansible_cmd(cmd => 'ls -lai /hana/', provider => $prov, filter => 'hana');
     my $cmr_status = qesap_ansible_script_output(cmd => 'crm status', provider => $prov, host => '"hana[0]"', root => 1);
+    die 'Error in stonith-sbd resource ' if cluster_check_status_by_regex($cmr_status, qr/stonith-sbd(.*?)(?=\n|$)/) == 1;
+    die 'Error in SAPHana_HDB resource ' if cluster_check_status_by_regex($cmr_status, qr/Clone Set:.*?\[(.*?)\].*?\* (\w+): \[/) == 1;
+    die 'Error in rsc_socat_HDB_HDB00 ' if cluster_check_status_by_regex($cmr_status, qr/rsc_socat_HDB_HDB00(.*?)(?=\n|$)/) == 1;
+    die 'Error in rsc_ip_HDB00 ' if cluster_check_status_by_regex($cmr_status, qr/Resource Group: (.*):\n  * (.*)/) == 1;
```

Comment:
I just noticed this regexp has `\n` as part of it, but in the code of `cluster_check_status_by_regex` you are splitting by lines, so I don't think any of the 3 regexps which have `\n` will ever work. Do you have an example of it working?

Reply:
refactored